Sources/SwiftDocC/Model/SourceLanguage.swift

```diff
@@ -135,6 +135,7 @@ public struct SourceLanguage: Hashable, Codable {
             "objective-c",
             "objc",
             "c", // FIXME: DocC should display C as its own language (github.com/apple/swift-docc/issues/169).
+            "c++",
```

Comment:
can we add "objective-c++" as well.

Reply:
If we do that, should we also add "occ+" and "objc++"?

Reply:
I've filed https://github.com/apple/swift-docc/issues/767 to track the issue and added it in a comment, alongside language aliases for various versions of Objective-C++.